README.md

```diff
@@ -23,6 +23,17 @@ To use the default template for the editable footer, add `src/addons/volto-edita
   ]
 ```
 
+### Configuration
+
+By default, column configuration shows all available options.
+If not needed, you could remove unused options in config:
+
+```json
+ config.settings["volto-editablefooter"] = {
```

Comment:
```suggestion
config.settings["volto-editablefooter"] = {
```
e questa indentazione no?

Reply:
i agree, but we will have to change our projects settings when we upgrade the addon